flake.nix

```diff
@@ -3,7 +3,9 @@
 
   inputs = {
     nixpkgsUnstable = {
-      url = "github:NixOS/nixpkgs/nixpkgs-unstable";
+      # TODO(msanft): Go back to upstream once the following PR lands:
+      # https://github.com/NixOS/nixpkgs/pull/395114
+      url = "github:msanft/nixpkgs/msanft/mkosi/fix";
```

Comment:
I noticed an open PR on mkosi for the issue. Why don't we wait for a version with the fix, instead of branching nixpkgs here?

Reply:
They won't do 25.4 anytime soon I think. But yes, we can wait for it to land.

Reply:
I'd like to at least wait for the nixpkgs PR.